UIDokan/Sources/UIDokan/ReusableViews/StarsView.swift

```diff
@@ -7,7 +7,19 @@
 
 import Cosmos
 import Foundation
+import UIKit
 
 /// Bridge to access `CosmosView` to avoid coupling in the project
 ///
-public class StarsView: CosmosView {}
+public class StarsView: CosmosView {
+
+    public func applyStyleToView(_ rate: Double) {
```

Comment:
Violates single responsibility. Let's create one for updating the rate and another for apply style.